packages/dds/tree/src/simple-tree/api/typesUnsafe.ts

```diff
@@ -225,15 +225,21 @@ export type InsertableTreeNodeFromImplicitAllowedTypesUnsafe<
 /**
  * {@link Unenforced} version of {@link InsertableTreeNodeFromAllowedTypes}.
  * @see {@link Input}
+ * @privateRemarks
+ * TODO: it seems like the order of the union this produces is what is non-deterministic in incremental builds
```

Comment:
What are the next steps here? Would be good to document that explicitly.

Reply:
I have no idea. Get lucky and have it stop breaking? Do a bunch of work to find a stable repro, minimize it and see if TypeScript already has an open bug for it? Turn off incremental build? See if it goes away after some random changes?

Reply:
We have had this known issue for months, and multiple failed attempts to make progress. Today was yet another failed attempt to figure it out, but I figured I'd at least document something while I'm here.

Reply:
If there isn't an action item, maybe it shouldn't be a `TODO`? To me "TODO" implies there is something actionable to do, even if that's just an investigation. Might be better to just say "NOTE: ...".

Reply:
I added a task to track this, since it looked like there wasn't one. Its actionable: someone can take the action of making a plan I guess.